APMrover2/test.cpp

```diff
@@ -7,23 +7,23 @@
 // User enters the string in the console to call the functions on the right.
 // See class Menu in AP_Common for implementation details
 static const struct Menu::command test_menu_commands[] = {
-	{"pwm",				MENU_FUNC(test_radio_pwm)},
-	{"radio",			MENU_FUNC(test_radio)},
-	{"passthru",		MENU_FUNC(test_passthru)},
-	{"failsafe",		MENU_FUNC(test_failsafe)},
-	{"relay",			MENU_FUNC(test_relay)},
-	{"waypoints",		MENU_FUNC(test_wp)},
-	{"modeswitch",		MENU_FUNC(test_modeswitch)},
-
-	// Tests below here are for hardware sensors only present
-	// when real sensors are attached or they are emulated
-	{"gps",			MENU_FUNC(test_gps)},
-	{"ins",			MENU_FUNC(test_ins)},
-	{"sonartest",	MENU_FUNC(test_sonar)},
-	{"compass",		MENU_FUNC(test_mag)},
-	{"logging",		MENU_FUNC(test_logging)},
+    {"pwm",             MENU_FUNC(test_radio_pwm)},
+    {"radio",           MENU_FUNC(test_radio)},
+    {"passthru",        MENU_FUNC(test_passthru)},
+    {"failsafe",        MENU_FUNC(test_failsafe)},
+    {"relay",           MENU_FUNC(test_relay)},
+    {"waypoints",       MENU_FUNC(test_wp)},
+    {"modeswitch",      MENU_FUNC(test_modeswitch)},
+
+    // Tests below here are for hardware sensors only present
+    // when real sensors are attached or they are emulated
+    {"gps",             MENU_FUNC(test_gps)},
+    {"ins",             MENU_FUNC(test_ins)},
+    {"sonartest",       MENU_FUNC(test_sonar)},
+    {"compass",         MENU_FUNC(test_mag)},
+    {"logging",         MENU_FUNC(test_logging)},
 #if CONFIG_HAL_BOARD == HAL_BOARD_PX4 || CONFIG_HAL_BOARD == HAL_BOARD_VRBRAIN
-    {"shell", 				MENU_FUNC(test_shell)},
+    {"shell",       MENU_FUNC(test_shell)},
```

Comment:
These lines always trying to hide from developers :smile: You missed this one :wink: